userspace/libsinsp/sinsp_filtercheck_thread.cpp

```diff
@@ -46,10 +46,10 @@ static inline bool str_match_start(const std::string& val, size_t len, const cha
 
 static const filtercheck_field_info sinsp_filter_check_thread_fields[] =
 {
-	{PT_CHARBUF, EPF_NONE, PF_NA, "proc.exe", "First Argument", "The first command line argument argv[0] (truncated after 4096 bytes) which is usually the executable name but it could be also a custom string, it depends on what the user specifies. This field is collected from the syscalls args or, as a fallback, extracted from /proc/PID/cmdline."},
+	{PT_CHARBUF, EPF_NONE, PF_NA, "proc.exe", "First Argument", "The first command-line argument (i.e., argv[0]), typically the executable name or a custom string as specified by the user. It is primarily obtained from syscall arguments, truncated after 4096 bytes, or, as a fallback, by reading /proc/PID/cmdline, in which case it may be truncated after 1024 bytes. This field may differ from the last component of proc.exepath, reflecting how command invocation and execution paths can vary."},
```

Comment:
Missing verb in the first sentence. "is typically"

Reply:
```
The first command-line argument (i.e., argv[0]), typically the executable name or a custom string as specified by the user, is primarily obtained from syscall arguments, truncated after 4096 bytes, or, as a fallback, by reading /proc/PID/cmdline, in which case it may be truncated after 1024 bytes. 
```

The sentence is long, but it works I would say.

Reply:
> The sentence is long, but it works I would say.

I, too, would have liked a shorter sentence, but I didn't find any alternative.

Reply:
👍